File: temp.c
#include<stdio.h>
#include<math.h>
int main() 
{
       int itemp, opt;
       float ftemp;
       printf(" 1. °C to K \n ");
       printf(" 2. °F to °C \n ");
       printf(" 3. °F to K \n ");
       printf(" 4. For reversal of 1st option \n ");
       printf(" 5. For reversal of 2nd option \n ");
       printf(" 6. For reversal of 3rd option \n ");
       scanf("%d", &opt);
       switch(opt)
    {
         case 1 : printf("Enter temp in °C : ");
                      scanf("%d", &itemp);
                      ftemp = itemp + 273 ;
                      printf("Temp in K is : %3.2f ", ftemp);
                      break;
         case 2 : printf("Enter temp in °F : ");
                      scanf("%d", &itemp);
                      ftemp = 1.8 * ( itemp + 32 ) ;
                      printf("Temp in °C is : %3.2f", ftemp);
                      break;
        case 3 : printf("Enter temp in °F : ");
                     scanf("%d", &itemp);
                     float ftok;
                     ftok = (itemp * 1.8) + 32;
                     ftemp = ftok + 273;
                     printf("Temp in K is : %3.2f", ftemp);
                     break;
       case 4 : printf("Enter temp in K : ");
                    scanf("%d", &itemp);
                    ftemp = itemp - 273 ;
                    printf("Temp in °C is : %3.2f", ftemp);
                    break;
      case 5 : printf("Enter temp in °C : ");
                   scanf("%d", &itemp);
                   ftemp = 0.55 * (itemp - 32) ;
                   printf("Temp in °F is : ~ %3.2f ", ftemp);
                   break;
      case 6 : printf("Enter temp in K : ");
                   scanf("%d", &itemp);
                   float ctok ;
                   ctok = itemp - 273 ;
                   ftemp = 0.55 * ( ctok - 32 );
                   break;
        default : printf(" Invalid option!!");
     }
        return 0;
}

File: temp.py
print(" 1. °C to K ")
print(" 2. K to °C  ")
print(" 3. °C to °F ")
print(" 4. °F to °C ")
print(" 5. K to °F ")
print(" 6. °F to K ")
op = input(" >>> ")
if op == "1" :
       num1 = float(input("Enter temp in °C  : "))
       result = str(num1 + 273 )
       print("Temp in K is : " +result)
elif op == "2" :
       num1 = float(input("Enter temp in K : "))
       result = str(num1 - 273 )
       print("Temp in °C is : " +result)
elif op == "3" :
       num1 = float(input("Enter temp in °C : "))
       result = str( ( num1 * 1.8) + 32 )
       print("Temp in °F is : " +result)
elif op == "4" :
       num1 = float(input("Enter temp. in °F : "))
       result = str( (num1 - 32) * (5 / 9) )
       print("Temp in °C is : " +result)
elif op == "5" :
       num1 = float(input("Enter temp in K : "))
       intermediate = num1 - 273 
       result = str( (intermediate * 1.8 ) + 32)
       print("Temp in °F is : " +result)
elif op == "6" :
       num1 = float(input("Enter temp in °F : "))
       intermediate = ( (num1 - 32) * (5 / 9) )
       result = str(intermediate + 273 )
       print("Temp in K is : " +result)
else : 
       print(" Invalid option !! ")